EasyPost/Exceptions/API/_Base.cs

```diff
@@ -88,7 +88,7 @@ internal static ApiError FromResponse(RestResponse response)
                 // could not extract error details from the API response (or API did not return data, i.e. 500) -> HttpError type rather than ApiError
                 errorMessage = response.ErrorMessage // fallback to standard HTTP error message
                                ?? response.StatusDescription // fallback to standard HTTP status description
-                               ?? "HTTP call did not return a standard error message"; // fallback to unknown error
+                               ?? "API did not return any error data"; // fallback to unknown error
```

Comment:
Thought: An argument could be made this should also live in that constants file?

Reply:
Smart thinking, resolved.